File: backend/src/main/java/bizworks/backend/dtos/AttendanceReportDTO.java
package bizworks.backend.dtos;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class AttendanceReportDTO {
    private String totalWorkTimeInWeek;
    private String totalWorkTimeInMonth;
    private String totalOvertimeInMonth;
}

File: mobile/mobile/lib/models/AttendanceReportDTO.dart
class AttendanceReportDTO {
  final String totalWorkTimeInWeek;
  final String totalWorkTimeInMonth;
  final String totalOvertimeInMonth;

  AttendanceReportDTO({
    required this.totalWorkTimeInWeek,
    required this.totalWorkTimeInMonth,
    required this.totalOvertimeInMonth,
  });

  factory AttendanceReportDTO.fromJson(Map<String, dynamic> json) {
    return AttendanceReportDTO(
      totalWorkTimeInWeek: json['totalWorkTimeInWeek'],
      totalWorkTimeInMonth: json['totalWorkTimeInMonth'],
      totalOvertimeInMonth: json['totalOvertimeInMonth'],
    );
  }
}
